tests/e2e/common_apps.go

```diff
@@ -150,7 +150,7 @@ func (td *OsmTestData) SimplePodApp(def SimplePodAppDef) (corev1.ServiceAccount,
 	return serviceAccountDefinition, podDefinition, serviceDefinition
 }
 
-// SimpleDeploymentAppDef defines some parametrization to create a deployment-based application from template
+// SimpleDeploymentAppDef defines some parametrization to create a pod-based application from template
```

Comment:
Did you mean to change this? Deployment makes more sense to me here.

Reply:
ouch
```suggestion
// SimpleDeploymentAppDef defines some parametrization to create a deployment-based application from template
```